vimrc.bundles

```diff
@@ -37,9 +37,10 @@ Plug 'jlanzarotta/bufexplorer', { 'commit': 'f3bbe12664b08038912faac586f6c0b5104
 Plug 'jparise/vim-graphql', { 'commit': '7ecedede603d16de5cca5ccefbde14d642b0d697' }
 Plug 'jtratner/vim-flavored-markdown'
 if expand('<sfile>') == '/etc/vim/vimrc.bundles'
-  Plug 'junegunn/fzf', { 'tag': '0.16.7', 'dir': '/etc/vim/fzf', 'do': './install --bin' }
+  Plug 'junegunn/fzf', { 'commit': 'ef80bd401fdab3979a72696dac585f1cbd463efc', 'dir': '/etc/vim/fzf', 'do': './install --bin' }
+
 else
-  Plug 'junegunn/fzf', { 'tag': '0.16.7', 'dir': '~/.fzf', 'do': './install --bin' }
+  Plug 'junegunn/fzf', { 'commit': 'ef80bd401fdab3979a72696dac585f1cbd463efc', 'dir': '~/.fzf', 'do': './install --bin' }
```

Comment:
Could you pin the latest release, 0.19.0, instead?  I think that's much cleaner than pinning arbitrary commits when it's an option.